### PR TITLE
Create ShivangiS04

### PR DESCRIPTION
**Pull request by ShivangiS04**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5880"></a>

### CODE REVIEW

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hi @ShivangiS04, thank you for contributing :tada:

File: data/ShivangiS04.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+"links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/aloobandar01",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Github",
+      "url": "https://twitter.com/ShivangiS04",
```
Comment (imbeshat) [inline data/ShivangiS04.json:14]: It should be Github URL.

File: data/ShivangiS04.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Shivangi Singh",
+  "type": "Personal",
+  "bio": "Just an Engineering undergrad trying to find her place in the world.",
+
+"links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/aloobandar01",
```
Comment (imbeshat) [inline data/ShivangiS04.json:9]: You have 2 different twitter url on line 9 & 14. Please, check which one is correct and use it here.

Comment (Pradumnasaraf) [reply data/ShivangiS04.json:9]: The fix can come in another PR. Thanks for the review

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Welcome to LinkFree 🎉

**Pull request merged**